pkgs/development/python-modules/netaddr/default.nix

```
@@ -4,33 +4,22 @@
 , pytest
 , fetchpatch
 , glibcLocales
+, pythonOlder
+, importlib-resources
 }:
 
 buildPythonPackage rec {
   pname = "netaddr";
-  version = "0.7.19";
+  version = "0.8.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "38aeec7cdd035081d3a4c306394b19d677623bf76fa0913f6695127c7753aefd";
+    sha256 = "0hx2npi0wnhwlcybilgwlddw6qffx1mb7a3sj4p9s7bvl33mgk6n";
   };
 
   LC_ALL = "en_US.UTF-8";
-  checkInputs = [ glibcLocales pytest ];
-
-  checkPhase = ''
-    # fails on python3.7: https://github.com/drkjam/netaddr/issues/182
-    py.test \
-      -k 'not test_ip_splitter_remove_prefix_larger_than_input_range' \
-      netaddr/tests
-  '';
-
-  patches = [
-    (fetchpatch {
-      url = "https://github.com/drkjam/netaddr/commit/2ab73f10be7069c9412e853d2d0caf29bd624012.patch";
-      sha256 = "0s1cdn9v5alpviabhcjmzc0m2pnpq9dh2fnnk2x96dnry1pshg39";
-    })
-  ];
+  checkInputs = [ glibcLocales pytest ]
+    ++ stdenv.lib.optionals (pythonOlder "3.7") [ importlib-resources ] ;
 
   meta = with stdenv.lib; {
     homepage = "https://github.com/drkjam/netaddr/";
```

```suggestion
    homepage = "https://netaddr.readthedocs.io/en/latest/";
    changelog = "https://netaddr.readthedocs.io/en/latest/changes.html";
```
That github repo is out of date.

Would you consider adding yourself as maintainer as well, since this package doesn't have a maintainer?

---

I updated the homepage to point to the `netaddr/netaddr` GitHub repo, which seems to be the new home.

I'd prefer not to add myself as a maintainer. I don't know much about this package other than it's in my nix-darwin system closure and doesn't build on current macOS without an update.

---

I generally prefer to point the homepage to what would make the most sense for users, not maintainers (i.e. documentation, not the repository), under the assumption that maintainers can find the repository themselves (though to be fair that's mostly for packages using ``fetchFromGitHub``, so the GitHub is obvious).

The ``nixpkgs`` manual also hints at this, by showing the GNU manual in ``homepage``. There's a not-used-enough ``meta.downloadPage`` attribute that the GitHub link would best fit in:
``downloadPage = "https://github.com/netaddr/netaddr/releases";``

---

> I generally prefer to point the homepage to what would make the most sense for users, not maintainers

Seems reasonable to me, I wasn’t aware of this convention but I do see a number of usages in the python part of nixpkgs. Updated to have the recommended `homepage` and `downloadPage`